test/math.test.ts

```diff
@@ -65,7 +65,7 @@ describe('Math properties tests', () => {
       .forall('a', fc.integer(-10, 10))
       .forall('b', fc.integer(-10, 10))
       .forall('c', fc.integer(-10, 10))
-      .then(({a, b, c}) => a + b * c === a + (b * c))
+      .then(({a, b, c}) => a + b * c === a + b * c)
       .check()
     ).to.have.property('satisfiable', true)
   })
```

Comment:
After this linting, I now realise this test is just stupid 😅 Ok to delete it?

Reply:
Yeah, I agree it does not make any sense now. I think it should be deleted.